evap/evaluation/views.py

```diff
@@ -80,3 +81,6 @@ def index(request):
 
 def faq(request):
     return render(request, "faq.html", dict(sections=FaqSection.objects.all()))
+
+def impressum(request):
+    return render(request, "impressum.html", dict(LEGAL_NOTICE_FILE=settings.LEGAL_NOTICE_FILE))
```

Comment:
is there a reason why this is done as a template parameter and teh other settings are not?